driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java

```diff
@@ -280,6 +280,7 @@ protected StringBuilder appendOptions(StringBuilder sb, boolean formatted) {
         if (cassandraVersion.getMajor() > 3 || (cassandraVersion.getMajor() == 3 && cassandraVersion.getMinor() >= 8)) {
             and(sb, formatted).append("cdc = ").append(options.isCDC());
         }
+		and(sb, formatted).append("memtable_flush_period_in_ms = ").append(options.getMemtableFlushPeriodInMs());
```

Comment:
This option has been introduced in Cassandra 2.0 by CASSANDRA-4237.

So i would suggest that you only include it when the version is 2.0 or higher:

```
        if (cassandraVersion.getMajor() > 1) {
            and(sb, formatted).append("memtable_flush_period_in_ms = ").append(options.getMemtableFlushPeriodInMs());
        }
```

Reply:
Thanks for the suggestion. Added suggested changes.